.env.staging

```
@@ -4,4 +4,4 @@ VITE_APP_FRONTEND_URL=https://staging.village.do
 VITE_APP_POSTHOG_HOST=https://app.posthog.com
 VITE_APP_POSTHOG_KEY=phc_eM9Ie4T0FvMBXIi5Dg0A9z6L2cT5Y0jY0zsJTQkYB6v
 
-VITE_WIDGET_DEV_PATH=D:/git/Village/admin/frontend/public/dev/village-widget.js
\ No newline at end of file
+VITE_WIDGET_DEV_PATH=/Users/ziad/projects/village/examples/authorize/public/village-widget.js
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)